chain/stmgr/stmgr.go

```diff
@@ -850,6 +850,7 @@ func (sm *StateManager) setupGenesisActors(ctx context.Context) error {
 			}
 
 		} else if act.Code == builtin.AccountActorCodeID {
+			// should exclude burnt funds actor and "remainder account actor"
```

Comment:
Does it do that?

Reply:
@magik6k #3047 will do the burnt part of it, the "remainder" part is TBD. But that's okay, because this method won't actually be used once #3136 goes in. 

We'll need to do the "remainder account" bit when we actually start using this method again.